source/userguide/active_learning.rst

```diff
@@ -0,0 +1,226 @@
+
+
+The following command can be used to run tuning using the settings defined in the application configuration:
+
+.. code-block:: console
+
+    mindmeld active_learning --tune --app-path '<PATH>/app_name/' --output_folder '<PATH>'
+
+Flags for application path and output folder are required and overwrite the default configuration settings for active learning. In addition to the aforementioned required flags, the following optional flags can be used - tuning_level, batch_size, n_epochs, train_seed_pct, and plot. These are described in detail in the AL config section.
+
+At the end of the tuning process, results are stored in the ``output_folder``. The ``accuracy.json`` file in the directory ``output_folder/results`` consist of strategy performance on the application's test/evaluation data for every iteration and epoch. ``selected_queries.json`` consists of the same information but instead of evaluation performance, this file records the queries selected at that iteration. the ``output_folder/plots`` directory consists of this quantitative information in a  visual format. The plots record performance of all chosen strategies across data iterations and gives a sense of which strategy is best suited for your application. The same information can be gauged from these results and plots about the best ``tuning_level`` for your application.
+
+
+Query Selection
+---------------
+
+Once the best performing strategy and level is known through tuning, the same set of hyperparameters can be carried over to the query selection step. Here, the active learning pipeline picks the best subset of queries from the logs that can be added to the training files to give the maximum performance boost in terms of accuracy.
+
+The following command can be used to run query selection using the settings defined in the application configuration if the log file or the log files' pattern has been specified in the config:
+
+.. code-block:: console
+
+    mindmeld active_learning --select --app-path '<PATH>/app_name/' --output_folder '<PATH>'
+
+
+Alternatively, path to unlabeled logs (``unlabeled_logs_path``) can be provided as a flag. 
+
+.. code-block:: console
+
+    mindmeld active_learning --select --app-path "<PATH>/app_name/" --output_folder '<PATH>' --unlabeled_logs_path "<PATH>/logs.txt"
+
+
+Also, if your log data is labelled and included in your MindMeld application you can specify the pattern for your log data using the following flag:
+
+.. code-block:: console 
+
+    mindmeld active_learning --select --app-path '<PATH>/app_name/' --output_folder '<PATH>' --labeled_logs_pattern ".*log.*.txt"
+
+Optional flags that can be used for selection include: ``batch_size``, ``log_usage_pct``, ``strategy``.
+
+
+.. note::
+
+    For labelled logs pattern, ensure that your log pattern (``labeled_logs_pattern``) does not overlap with your train pattern (``train_pattern``).
```

Comment:
Would be helpful to have a quick example here of what we mean by "overlapping pattern"